app/src/main/java/org/commcare/commcaresupportlibrary/identity/model/MatchStrength.java

```diff
@@ -0,0 +1,14 @@
+package org.commcare.commcaresupportlibrary.identity.model;
+
+public enum MatchStrength {
+    // Almost certain match
+    CERTAIN,
```

Comment:
Honestly it might be fine to just articulate this by what it's used for ("One Star" "Two Star"), since if I were a biometrics person it would make me anxious to ever use "CERTAIN" as a field, and I wouldn't want the result to be misinterpreted